_delphi_utils_python/pyproject.toml

```diff
@@ -0,0 +1,59 @@
+[build-system]
+requires = ["setuptools>=70.1", "setuptools-scm>=8.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "delphi-utils"
+version = "0.3.24"
+description = "Shared Utility Functions for Indicators"
+readme = "README.md"
+requires-python = "== 3.8.*"
+license = { text = "MIT License" }
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Programming Language :: Python :: 3.8",
+    "License :: MIT",
```

Comment:
the curse of my past that caused error in the release back in september "License :: OSI Approved :: MIT License"